cms/static/js/views/pages/paged_container.js

```diff
@@ -29,8 +31,14 @@ define(["jquery", "underscore", "gettext", "js/views/pages/container", "js/views
                 } else {
                     this.refreshChildXBlock(xblockElement, block_added);
                 }
-            }
+            },
 
+            toggleChildrenPreviews: function(xblockElement) {
```

Comment:
This does not call `event.preventDefault()` - as a result the page scrolls and the `#` appears in the URL when the button is clicked.